lib/env_irb_prompt.rb

```diff
@@ -0,0 +1,36 @@
+require 'socket'
+
+class EnvIrbPrompt
+  # @param [Hash] irb_conf
+  def configure!(irb_conf = IRB.conf)
+    irb_conf[:USE_AUTOCOMPLETE] = false
+    irb_conf[:SAVE_HISTORY] = on_deployed_box? ? nil : 1000
+
+    irb_conf[:PROMPT][:ENV_PROMPT] = {
+      PROMPT_I: "%N(#{bold(env)}):%03n:%i> ",
+      PROMPT_S: "%N(\e[1m#{bold(env)}\e[22m):%03n:%i%l ",
+      PROMPT_C: "%N(\e[1m#{bold(env)}\e[22m):%03n:%i* ",
```

Comment:
Is it not possible to use `Identity::Hostdata.env` here?

Reply:
That requires `bundler` to be loaded, so this work with plain `irb`